locator/token_metadata.cc

```diff
@@ -535,15 +536,14 @@ class token_metadata_impl final {
 
 #endif
     std::vector<inet_address> get_all_endpoints() const {
-        std::vector<inet_address> tmp;
-        std::transform(_endpoint_to_host_id_map.begin(), _endpoint_to_host_id_map.end(), std::back_inserter(tmp), [](const auto& p) {
-           return p.first;
-        });
-        return tmp;
+        std::unordered_set<inet_address> tmp;
+        boost::copy_range<std::unordered_set<gms::inet_address>>(_token_to_endpoint_map | boost::adaptors::map_values);
+        std::vector<inet_address> ret(tmp.begin(), tmp.end());
+        return ret;
```

Comment:
How about simplifying this to `return std::vector<inet_address>(tmp.begin(), tmp.end());`?

Reply:
yes